docs/api-docs/catalog/v2-v3-examples.md

```diff
@@ -0,0 +1,1052 @@
+        "resource": "/products/77/skus"
+    },
+    "rules": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/rules",
+        "resource": "/products/77/rules"
+    },
+    "option_set": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/optionsets/14",
+        "resource": "/optionsets/14"
+    },
+    "options": {
+        "url": "https://api.bigcommerce.com/stores/{store_hash}/v2/products/77/options",
+        "resource": "/products/77/options"
+    },
+    ...
+    },
+    "metadata": []
+}
+```
+
+### Create a product
+
+To create a simple product using the V3 Catalog API, send a `POST` request to `/v3/catalog/products`.
+
+**V3 request**
+
+`POST /v3/catalog/products`
+
+```json
+{
+  "name": "BigCommerce Storage Basket",
+  "price": "10.00",
+  "categories": [
+    23,
+    21
+  ],
+  "weight": 4,
+  "type": "physical"
+}
+```
+
+To create a simple product using the V2 Products API, in addition to the properties in the V3 example, your `POST` request must include `availability`.
+
+**V2 request**
+
+`POST /v2/products`
+
+```json
+{
+  "name": "BigCommerce Clay Vase",
+  "price": "20.00",
+  "categories": [
+    23,
+    21
+  ],
+  "weight": 5,
+  "type": "physical",
+  "availability": "available"
+}
+```
+
+### Create a product with images
+
+To create a simple product with an image using the V3 Catalog API, first create the product, then add the image.
+
+**V3 request**
+
+`POST /v3/catalog/products/{product_id}/images`
+
+1. Create a product using `POST /v3/catalog/products`
+2. Add an image using `POST /v3/catalog/products/{product_id}/images`
+
+```json
+{
+  "is_thumbnail": true,
+  "sort_order": 1,
+  "description": "Top View",
+  "image_url": "https://upload.wikimedia.org/wikipedia/commons/7/7f/Anglel_Bless_Legendary_Hills_1_m%C4%9Bs%C3%ADc_st%C3%A1%C5%99%C3%AD.jpg"
+}
+```
+
+Creating a product with an image in V2 follows a similar sequence; however, the V2 `/images` endpoint only accepts the `multipart/form-data` media type.
+
+**V2 request**
+
+`POST /v2/products/{product_id}/images`
+
+1. Create a product using `POST /v2/products`
+2. Add an image using `POST /v2/products/{product_id}/images`
+
+```shell
+curl -X POST \
+  https://api.bigcommerce.com/stores/{store_hash}/v2/products/{product_id}/images \
+  -H 'Accept: application/json' \
+  -H 'Cache-Control: no-cache' \
+  -H 'Content-Type: application/json' \
+  -H 'Postman-Token: 841f5f9a-244b-4d2c-900f-938ac2067a4a' \
+  -H 'X-Auth-Token: {X-Auth-Token}' \
+  -H 'content-type: multipart/form-data; boundary=-WebKitFormBoundary7MA4YWxkTrZu0gW' \
+  -F image_file=@/Users/{user_name}/Documents/product_images/image_file.png
+  ```
+
+### Create a product with videos
+
+To create a simple product with an video using the V3 Catalog API, first create the product, then add the video. All videos must be loaded through YouTube and have a `video_id`.
```

Comment:
Same as with adding images above, this might not need to be a series since you've already outlined product creation above.